### ISSUE TITLE
[localize] formatNumber for en-AU is broken with currency sign

### ISSUE THREAD

**Issue by daKmoR** [ing-bank/lion#432]
## Describe the bug

these tests fail 

```js
expect(formatNumber(123456.789, currencySymbol('EUR'))).to.equal('€123,456.79');
expect(formatNumber(123456.789, currencySymbol('USD'))).to.equal('$123,456.79');
expect(formatNumber(123456.789, currencySymbol('JPY'))).to.equal('¥123,457');
```

**Comment by Guddu327** [comment]
Hi..!! I am new to open source and I would like to work on this, if no one else is currently working on this..

**Comment by Guddu327** [comment]
I tried looking into the issue and found out the formatted number returned by `Intl.NumberFormat`, inside the function `formatNumberToParts`, is inconsistent for the mentioned test cases of `en-AU` locale, as compared to similar test cases used with other locales. I'm unsure of how to proceed with this now, since `Intl` is a web api.

**Comment by daKmoR** [comment]
hey, there and welcome to open source 🤗 

web apis are made by people too and mistakes happen 🙈 

that's why we have localize so we are able to normalize browser bugs/inconsistencies...
we already have multiple such "fixes":
- https://github.com/ing-bank/lion/blob/master/packages/localize/src/number/forceCurrencyToEnd.js
- https://github.com/ing-bank/lion/blob/master/packages/localize/src/number/forceAddGroupSeparators.js

basically every file in that starts with `force` is a workaround for some "bad" responses from `Intl`.

So best would be to add these tests, let them fail and then start adding a `forceCurrencyForEnAu` or something in the likes.

does that sound like a plan?

**Comment by Guddu327** [comment]
Hey thank you for the suggestions.
I was able to add some changes and added some new scripts which modifies the `normalizeInt` script.

I am having some difficulties in adding the pull requests. I have done till the last step by following `Contribution.md`, but while adding the pull request, it's showing 'There isn’t anything to compare.'.

**Comment by daKmoR** [comment]
fixed via https://github.com/ing-bank/lion/pull/619
